.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: "CI"
+
+on:
+  pull_request:
+  push:
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    name: "Test on PHP ${{ matrix.php-version }}"
+
+    runs-on: 'ubuntu-latest'
+
+    continue-on-error: true
+
+    strategy:
+      matrix:
+        php-version:
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0'
+          - '8.1'
+          - '8.2'
+
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v3
+
+      - name: "Install PHP with extensions"
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "none"
+          php-version: ${{ matrix.php-version }}
+          ini-values: memory_limit=-1
+
+      - name: "Add PHPUnit matcher"
+        run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
```

Comment:
What does this do?

Reply:
Guessing here, but since we execute `simple-phpunit` instead of running phpunit directly, GH actions doesn't know how to map errors back to code to show properly in the action results. This fixes that by referring to the gh-action built-in mapping configuration for phpunit.  https://fusectore.dev/2021/11/19/github-action-problem-matchers.html